pkg/healthcheck/microshift_core_workloads.go

```diff
@@ -30,15 +32,76 @@ func getCoreMicroShiftWorkloads() (map[string]NamespaceWorkloads, error) {
 			},
 		},
 	}
-	fillOptionalWorkloadsIfApplicable(cfg, workloads)
+	if cfg.Network.IsEnabled() {
+		workloads["openshift-ovn-kubernetes"] = NamespaceWorkloads{
+			DaemonSets: []string{"ovnkube-master", "ovnkube-node"},
+		}
+	}
+	if err := fillOptionalWorkloadsIfApplicable(cfg, workloads); err != nil {
+		return nil, err
+	}
 
 	return workloads, nil
 }
 
-func fillOptionalWorkloadsIfApplicable(cfg *config.Config, workloads map[string]NamespaceWorkloads) {
-	klog.V(2).Infof("Configured storage driver value: %q", string(cfg.Storage.Driver))
-	if cfg.Storage.IsEnabled() {
-		klog.Infof("LVMS is enabled")
+func lvmsIsExpected(cfg *config.Config) (bool, error) {
+	cfgFile := "/etc/microshift/lvmd.yaml"
+	if exists, err := util.PathExists(cfgFile); err != nil {
+		return false, err
+	} else if exists {
+		klog.Infof("%s exists - expecting LVMS to be deployed", cfgFile)
+		return true, nil
+	}
+
+	if !cfg.Storage.IsEnabled() {
+		klog.Infof("LVMS is disabled via config. Configured value: %q", string(cfg.Storage.Driver))
+		return false, nil
+	}
+
+	cmd := exec.Command("vgs", "--readonly", "--options=name", "--reportformat=json")
+	output, err := cmd.Output()
+	if err != nil {
+		return false, err
+	}
+	klog.V(2).Infof("vgs reported: %s", strings.ReplaceAll(strings.ReplaceAll(string(output), "\n", ""), " ", ""))
```

Comment:
Should we use https://pkg.go.dev/encoding/json#Compact instead?

Reply:
Wasn't aware of it, thx